frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -84,6 +104,45 @@ export default class Metadata extends React.Component<Props> {
         </Localized>;
     }
 
+    renderGroupComment(entity: Entity): React.Node {
+        if (!entity.group_comment) {
+            return null;
+        }
+
+        return <Localized id='entitydetails-Metadata--group-comment' attrs={ { title: true } }>
+            <Property title='Group Comment' className='comment'>
+                <Linkify properties={ { target: '_blank', rel: 'noopener noreferrer' } }>
+                    { entity.group_comment }
+                </Linkify>
+            </Property>
+        </Localized>;
+    }
+
+    renderResourceComment(entity: Entity): React.Node {
+        const { seeMore } = this.state;
+
+        if (!entity.resource_comment) {
+            return null;
+        }
+
+        let comment = entity.resource_comment
+
+        return <Localized id='entitydetails-Metadata--resource-comment' attrs={ { title: true } }>
+            <Property title='Resource Comment' className='comment truncate'>
+                <Linkify properties={ { target: '_blank', rel: 'noopener noreferrer' } } className='resource'>
+                    { comment.length < 85 || seeMore ? comment : comment.slice(0, 85) + "..." }
```

Comment:
Instead of "...", we should use the https://en.wikipedia.org/wiki/Ellipsis (…) -> it's a single character.